spec/services/accessories_child_updater_spec.rb

```diff
@@ -63,7 +63,7 @@
       end
 
       it "moves the child to canceled" do
-        expect(child_order_detail).to be_canceled
+        expect(child_order_detail.state).to eq "canceled"
```

Comment:
I slightly lean towards the older syntax, but only slightly. Is this related to the `is_canceled?` method we've added?